lib/views/course_view/downloaded_pinned_courses_view/content_view.dart

```diff
@@ -15,18 +15,21 @@ import 'package:gocast_mobile/views/video_view/video_card_view.dart';
 class CourseContentScreen extends ConsumerWidget {
   final String title;
   final List<VideoCard> videoCards;
+  final Future<void> Function()? onRefresh;
```

Comment:
`onRefresh` is not needed anymore since we use the following in all views:
```dart
body: RefreshIndicator(
  onRefresh: () async {
    await ref.read(userViewModelProvider.notifier).fetchUserPinned();
  },
```

Reply:
✅